app/commands.py

```diff
@@ -344,3 +347,24 @@ async def match_start_handler(session: "Session", args: list[str]) -> str | None
             session_id,
             data=match_started_packet,
         )
+
+
+@command("!silence", privileges=ServerPrivileges.CHAT_MODERATOR)
+async def silence_handler(session: "Session", args: list[str]) -> str | None:
+    duration = " ".join(args)
+
+    try:
+        raw_seconds = parse(duration)
+    except:
+        return "Invalid time format! Try again!"
```

Comment:
parse() does not actually raise an exc, instead it returns None upon error; remove try/catch check for none instead